docs/build_version_doc/build_all_version.sh

```diff
@@ -59,27 +65,8 @@ for tag in $tag_list; do
     make clean
     cd docs
     make clean
-    make html USE_OPENMP=0 || exit 1
-    python build_version_doc/AddVersion.py --file_path "_build/html/" --current_version "$tag" || exit 1
-
-    if [ $tag != 'master' ]
-    then 
-        python build_version_doc/AddPackageLink.py --file_path "_build/html/get_started/install.html" \
-                                                   --current_version "$tag" || exit 1
-    fi
-
-    if [ $version_num == 0 ]
-    then
-        cp -a _build/html/. "../../$built"
-    else
-        file_loc="../../$built/versions/$tag"
-        mkdir "$file_loc"
-        cp -a _build/html/. "$file_loc"
-    fi
+    make html USE_OPENMP=1 || exit 1
 
     ((++version_num))
 done
-    
-mv "$tag_file" "../../$built/tag.txt"
-cd ../..
-rm -rf "$mxnet_folder"
+
```

Comment:
Please add a comment that update_all_versions.sh should be run afterwards. Maybe even print it?

Reply:
Added.